### ISSUE TITLE
07 strings: Skip docstrings?

### ISSUE THREAD

**Issue by danmichaelo** [scriptotek/programming-for-lawyers#11]
Google Colab seems to have default indentation set to 2 characters, so a learner got indentation error from an attempted solution like this:

```python
def invert_name(name):
    """
    Convert a name from regular to inverted form
    >>> invert_name('John Doe')
    'Doe, John'
    """
  name_parts = name.split(" ")
  name_parts.reverse()
  reversed_name = ", ".join(name_parts)
  return reversed_name

print(invert_name("John Doe"))
```

Also, we didn't really introduce the concept of docstrings or documentation, and I think there are several aspects of docstrings that could be potentially confusing, like: Why is there Python code inside there at all? What is the meaning of `>>> `, and so on.

**Comment by ewinge** [comment]
Fixed in d9d590aa1e81d8f278a43e6ae6a4fa1e6a47780f

**Comment by ewinge** [comment]
indentation can be changed:
https://stackoverflow.com/questions/54971245/why-does-google-colab-uses-2-spaces-per-indentation-level-and-how-to-change-th
